test/unit/math/p5.Vector.js

```diff
@@ -1882,5 +1882,48 @@ suite('p5.Vector', function() {
         expect(p5.Vector.equals(a1, a2)).to.be.true;
       });
     });
+
+    suite('p5.Vector.clampToZero()', function() {
+      let v;
+
+      test('should clamp very small positive number of vector components to zero', function() {
+        v = new p5.Vector(0.0000000000000002, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0);
+        expect(v.y).to.equal(5);
+      });
+
+      test('should clamp very small negative number of vector components to zero', function() {
+        v = new p5.Vector(-0.0000000000000002, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0);
+        expect(v.y).to.equal(5);
+      });
+
+      test('should not clamp regular numbers of vector components', function() {
+        v = new p5.Vector(0.01, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0.01);
+        expect(v.y).to.equal(5);
+      });
```

Comment:
Here as well, it looks like we are checking for the same test and we are specifically testing for `Number.EPSILON` . What I think we can remove all the tests and can just keep a single big test as you have a test at last. 

something with :
```
        v = new p5.Vector(
          0.00000000000000005,
          -0.0000000000000002220446049250313,
          0.0000000000000002220446049250313);
```
With these three values, we can create different combinations to verify if clampToZero() works correctly. Since unit tests can be time-consuming to run, this would be a kind of optimization.

Let me know what you think ;)